tests/test_backends.py

```
@@ -0,0 +1,18 @@
+import pytest
+
+from phaseshifts.backends import EEASiSSSBackend, VHTBackend, get_backend
+
+
+def test_get_backend_default():
+    backend = get_backend(None)
+    assert isinstance(backend, VHTBackend)
+
+
+def test_get_backend_unknown():
+    with pytest.raises(ValueError):
```

Ruff (0.14.10)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)